gateway/1.4/modules/ROOT/pages/flex-gateway-k8-change-helm-settings.adoc

```diff
@@ -0,0 +1,231 @@
+registration:
+  ...
+resources:
+  limits:
+    cpu: 1000m
+    memory: 2024Mi
+  requests:
+    cpu: 250m
+    memory: 256Mi
+----
++
+This example displays settings from `REVISION` number `2`. The output displays only revised settings. If your chart has never revised the default Helm chart values, _no_ revisions appear in the output. 
+
+. Use one of the following procedures to change settings in the Helm chart for your Flex Gateway deployment: 
+
+* <<use-helm-set-option>> uses `--set` options in a `helm upgrade` command. The command allows multiple `--set` options. 
+* <<use-helm-f-option>> passes a configuration file with multiple settings though the `-f` option to `helm upgrade`.
+
+[[use-helm-set-option]]
+== Modify Settings for Your Helm Chart
+
+After following the steps in <<update-cpu-memory-example>>, use the `--set` option to pass new values individually to the Helm chart for your Flex Gateway deployment. Alternatively, follow the procedure in <<use-helm-f-option>>.
+
+. From the command line, navigate to the directory that contains the YAML registration file for your gateway release, such as a `registration.yaml` for an `ingress` release.
++
+This file is generated when you register your gateway with Anypoint Platform in xref:flex-conn-reg-run.adoc[Connected Mode] or xref:flex-local-reg-run.adoc[Local Mode]. The file provides `registration.content` data for the `helm upgrade` step in this procedure. 
+
+. Change the `cpu` and `memory` values by using the `helm upgrade` command.
++
+.Syntax:
+----
+helm upgrade <release-name> <repository name>/<chart name> \
+-i --wait -n <namespace> --create-namespace \
+--set-file registration.content=registration.yaml \
+--set <key-to-update>=<new-value> 
+--set <next-key-to-update>=<new-value> 
+...
+----
++
+The `helm upgrade` command relies on the `registration.yaml` file but does not add your new settings to this file. 
++
+.Example:
+[source,helm]
+----
+helm upgrade ingress flex-gateway/flex-gateway \
+-i --wait -n gateway --create-namespace \
+--set-file registration.content=registration.yaml \
+--set resources.limits.cpu=1500m \
+--set resources.limits.memory=2048Mi \
+--set resources.requests.cpu=260m \
+--set resources.requests.memory=384Mi
+----
++
+When successful, the command returns output similar to this:
++
+----
+Release "ingress" has been upgraded. Happy Helming!
+NAME: ingress
+LAST DEPLOYED: Mon Mar 27 11:58:36 2023
+NAMESPACE: gateway
+STATUS: deployed
+REVISION: 3
+TEST SUITE: None
+----  
+
+. Using the new `REVISION` value, check the `resources` settings, for example:
++
+[source,helm]
+----
+helm get values --revision=3 ingress -n gateway
+----
++
+The output displays the new `resources` settings for the Pod:
++
+----
+USER-SUPPLIED VALUES:
+registration:
+  ...
+resources:
+  limits:
+    cpu: 1500m
+    memory: 2048Mi
+  requests:
+    cpu: 260m
+    memory: 384Mi
+----
++
+Note that the `resources` settings are not displayed through the `registration.yaml`.
```

Comment:
Suggestion: remove "Note that"
We try not to use admonitions unless they are really needed.